cab-aggregator-request-response-logger-spring-boot-starter/src/main/java/com/modsen/cabaggregatorrequestresponseloggerspringbootstarter/logger/RequestResponseLogger.java

```diff
@@ -84,7 +133,12 @@ private String convertObjectToJson(Object object) {
         if (object == null)
             return "";
         try {
-            return objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(object);
+            if (object instanceof @SuppressWarnings("rawtypes") ResponseEntity re
+                    && re.getBody() instanceof InputStreamResource) {
+                return "";
+            }
```

Comment:
Посмотри такой вариант проверки:
```suggestion
            if (object instanceof ResponseEntity<InputStreamResource> re) {
                return "[Input stream]";
            }
```
Для данного случая можно вернуть более информативную строку, чтобы не перепутать с тем случаем, когда object == null.